apps/api/mix.exs

```diff
@@ -20,7 +20,7 @@ defmodule API.Mixfile do
   # Type `mix help compile.app` for more information.
   def application do
     [mod: {API.Application, []},
-     extra_applications: [:db, :logger, :guardian, :runtime_tools]]
+     extra_applications: [:db, :logger, :guardian, :runtime_tools, :verk]]
```

Comment:
we shouldn't need this since we're 1.4+, in fact I don't think we need `db` or `guardian` either

Reply:
we need it for when we move to distillery. its how distillery knows which applications to start on entry